src/concrete/ml/sklearn/tree_to_numpy.py

```diff
@@ -33,7 +33,7 @@
 # pylint: enable=wrong-import-position,wrong-import-order
 
 # Most significant bits to retain when applying rounding to the tree
-MSB_TO_KEEP_FOR_TREES = 1
+MSB_TO_KEEP_FOR_TREES = 4
```

Comment:
I would keep 4 here for now.

Reply:
I don't control the `msb=4`, I'm getting a lot of overflows.

I would suggest to keep msb=1 with this workaround. Then, change it to 4, when we go to truncate.